tests/tekton-resources/tasks/generators/clusterloader/sustained-scheduler-throughput.yaml

```diff
@@ -0,0 +1,158 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sustained-scheduler-throughput
+  namespace: scalability
+spec:
+  description: "clusterloader2 task to run various types of cl2 tests on a given cluster."
+  params:
+  - name: giturl
+    description: "git url to clone the package"
+    default: https://github.com/dheeraj-coding/perf-tests.git
```

Comment:
Let's not use individual forks please. We should be using upstream perf-tests repo, if it can't be checked into upstream, you can add it to assets folder, for example - [xref](https://github.com/awslabs/kubernetes-iteration-toolkit/blob/3f9489a39046ba9ef10426ccd244bd4c1e578a27/tests/assets/aiml-workload/large-sts/config.yaml)